cmd/print-follow-graph/print-follow-graph.go

```diff
@@ -20,13 +21,20 @@ func check(err error) {
 
 func main() {
 	var args generation.Args
-	flag.StringVar(&args.FixturesRoot, "fixtures", "./fixtures-output", "root folder containing spliced out ssb-fixtures")
 	flag.IntVar(&args.MaxHops, "hops", 2, "the max hops count to use")
 	flag.IntVar(&args.FocusedCount, "focused", 2, "number of puppets to use for focus group (i.e. # of puppets that verify they are replicating others)")
 	flag.Parse()
+	if len(flag.Args()) == 0 {
+		fmt.Printf("print-follow-graph <options> path-to-spliced-fixtures\n")
+		fmt.Println("prints the follow graph described by a spliced out fixtures folder (netsim generate output), and its replication expectations")
+		fmt.Println("Options:")
+		flag.PrintDefaults()
+		os.Exit(1)
+	}
+	args.FixturesRoot = flag.Args()[0]
 
 	var err error
-	g := generation.Generator{Args: args}
+	g := generation.Generator{Args: args, Output: os.Stdout}
```

Comment:
just as a general thought: If you want to protect against misuse, don't export the fields and expose a `func NewGenerator()` with explicit arguments that have to be passed. This will also allow you to default `output = os.Stderr` if it's nil for instance.

Reply:
good point, but i'll skip this atm. bringing with this knowledge for future decisions :~